NativeApp/src/components/Common/HeaderDays/styles.js

```diff
@@ -1,45 +1,37 @@
 import { StyleSheet } from 'react-native';
-import {LIGHTGREY, GREY, DARKGREY } from '../../../styles/colors';
+import { WHITE, GREY, LIGHTGREY } from '../../../styles/colors';
 
 const styles = StyleSheet.create({
   headerContainer: {
     flex: 1,
     flexDirection: 'row',
-    backgroundColor: LIGHTGREY,
-    paddingTop: 8,
-    paddingBottom: 8,
-    maxHeight: 93,
-    justifyContent: 'space-between',
-    borderColor: GREY,
-    borderBottomWidth: 1,
-    alignSelf: 'center',
-    alignItems: 'center',
+    backgroundColor: WHITE,
+    paddingTop: 11,
+    paddingBottom: 11,
+    maxHeight: 70,
+    borderColor: LIGHTGREY,
+    borderTopWidth: 0.7,
   },
   holidayContainer: {
-    alignItems: 'center',
+    marginLeft: '5%',
```

Comment:
Is there a reason for using a percentage for margin left? I'd rather if you used `justifyContent` or something similar to do with flex. Then add a number for marginLeft, e.g `marginLeft: 3`.

I'm not a big fan of using percentage for containers as it might look off on different devices. Especially devices with wider screens.

Reply:
i have changed the marginLeft to `marginLeft: 20`